code/game/machinery/cloning.dm

```diff
@@ -193,7 +193,7 @@
 		return ERROR_MISSING_EXPERIMENTAL_POD
 
 	if(!body_only && !(experimental && experimental_pod))
-		clonemind = locate(mindref) in SSticker.minds
+		clonemind = mindref
```

Comment:
`mindref` is a wrong name because it's not ref. I suggest `given_mind` or something.
(god, why clonemind is a var...)